tests/publiccloud/azure_cli.pm

```diff
@@ -17,6 +17,8 @@ use mmapi 'get_current_job_id';
 use utils qw(zypper_call script_retry);
 use version_utils 'is_sle';
 use registration qw(add_suseconnect_product get_addon_fullname);
+use publiccloud::ssh_interactive 'select_host_console';
```

Comment:
it is not used , please remove it

Reply:
done

Reply:
you writing "done" but code is still here